opc_da/src/traits/server.rs

```diff
@@ -0,0 +1,19 @@
+pub trait ServerTrait {
+    // IOPCCommon
+    fn set_locale_id(&self, locale_id: u32) -> windows_core::Result<()>;
+
+    // IOPCCommon
+    fn get_locale_id(&self) -> windows_core::Result<u32>;
+
+    // IOPCCommon
+    fn query_available_locale_ids(&self) -> windows_core::Result<Vec<u32>>;
+
+    // IOPCCommon
+    fn get_error_string(&self, error: i32) -> windows_core::Result<String>;
+
+    // IOPCCommon
+    fn set_client_name(&self, name: String) -> windows_core::Result<()>;
+
```

Comment:
why must constrain the client name length less than 255?

Reply:
Also I prefer to only declare methods in ServerTrait which wraps the Com Interface. So I can check the name in IOPCCommon_Impl::SetClientName. And I think windows_core::Error::from_win32 is not correct because i should create the specific error